### PR TITLE
FIM Scenarios config unification

### PR DESCRIPTION
**Pull request by Zenidd**
Hi team, 

This PR unifies the current scenarios configurations files for windows and linux agents. We now use `recursion_level=320` and `check_all="yes"` as default. 

Greetings, JP Sáez

### CODE REVIEW

File: tests/system/fim/scenarios/204_whodata_linux_noaudit/config/agent_windows_ossec.conf [hunk 1]
```diff
@@ -72,9 +72,9 @@
 
     <!-- Frequency that syscheck is executed default every 12 hours -->
     <frequency>10</frequency>
```
Comment (jm404) [inline tests/system/fim/scenarios/204_whodata_linux_noaudit/config/agent_windows_ossec.conf]: Please set it to 43200

File: tests/system/fim/scenarios/203_whodata_frequency/config/agent_windows_ossec.conf [hunk 2]
```diff
@@ -72,9 +72,9 @@
 
     <!-- Frequency that syscheck is executed default every 12 hours -->
     <frequency>43200</frequency>
```
Comment (jm404) [inline tests/system/fim/scenarios/203_whodata_frequency/config/agent_windows_ossec.conf]: Please check the linux analog config

File: tests/system/fim/scenarios/204_whodata_linux_noaudit/config/agent_windows_ossec.conf (hunk 1 above)
Comment (Zenidd) [reply tests/system/fim/scenarios/204_whodata_linux_noaudit/config/agent_windows_ossec.conf]: Fixed !

File: tests/system/fim/scenarios/203_whodata_frequency/config/agent_windows_ossec.conf (hunk 2 above)
Comment (Zenidd) [reply tests/system/fim/scenarios/203_whodata_frequency/config/agent_windows_ossec.conf]: Checked, both are correctly set to 43200.

Cheers !

**Pull request merged**